tests/viz/test_exporter.py

```diff
@@ -749,3 +744,670 @@ def run_simulation_save_results(units: pp.Units, file_name: str):
             num_vtk_tested += 1
 
     assert num_vtk_tested > 0, "No VTU files were tested."
+
+
+def _polytop_grid_2d() -> pp.Grid:
```

Comment:
I suggest putting the functions into src/porepy/applications/test_utils/grids.py.